Makefile.win

```diff
@@ -0,0 +1,41 @@
+!IF [where /R priv /Q Makefile.win.auto]
```

Comment:
I am completely ignorant of building C/C++ project on Windows without CMake - what generates the `.auto` file here, and what does it contain?

Reply:
I'm not much less ignorant for sure, and before this journey, probably just as :)

The makefile generates it, it's a filthy hack I saw at the argon2 makefile. It's needed because nmake does not let you set a variable from a subshell call. I tried what you mention in the other comment https://github.com/bitwalker/picosat_elixir/pull/3#discussion_r498541412 and this works, so the makefile can be much less complex